server/src/controllers/pollController.ts

```diff
@@ -58,14 +58,25 @@ async function changePollStatus(pollId: string, hasStarted: boolean) {
   return { status: 200, data: { message: "poll status successfully changed" } };
 }
 
+type AggregatedStudent = {
+  pollId: string,
+  courseCode: string,
+  sequence: number,
+  utorid: string,
+  timestamp: string,
+  pollName: string,
+  description: string,
+  answer: number
+}
+
```

Comment:
Create a new folder `types` under `server/src`. Shift this type to `server/src/types/pollController.types.ts`
Update the project structure on readme.md as well

Reply:
Fixed in 45617f3